### PR TITLE
Arquillian-cube tests, license header

### PR DESCRIPTION
**Pull request by pavolloffay**
The same test as for k8s https://github.com/jaegertracing/jaeger-kubernetes/pull/2.

- [x] travis
- [x] mvnw
- [x] license
- [x] test: request to UI

### CODE REVIEW

**Comment by pavolloffay** [comment]
@jpkrohling @objectiser could you please review. I tested it with minishift and `oc cluster up` everything worked well.

File: all-in-one/pom.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <artifactId>jaegertracing-openshift-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <rootProject.basedir>${project.basedir}/..</rootProject.basedir>
+    <template.name>jaeger-all-in-one-template.yml</template.name>
+  </properties>
+
+  <artifactId>jaegertracing-opeshift-all-in-one</artifactId>
```
Comment (objectiser) [inline all-in-one/pom.xml]: openshift

File: all-in-one/pom.xml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.jaegertracing.openshift.test</groupId>
```
Comment (objectiser) [inline all-in-one/pom.xml]: Wondering why the `test` suffix on the group?

Comment (pavolloffay) [inline all-in-one/pom.xml]: I don't know.. Should I remove it? These artifacts are not going to be published, at least not now.

**Comment by pavolloffay** [comment]
One last thing, I will put tests in a profile as they need `oc` command installed and travis is failing because of it. (The same for k8s to be consistent)

File: all-in-one/pom.xml (hunk 2 above)
Comment (objectiser) [inline all-in-one/pom.xml]: I think it would be better to remove.

**Comment by pavolloffay** [comment]
Updated, could you please have last quick look and merge if it's fine?

File: README.md [hunk 3]
```diff
@@ -23,3 +23,13 @@ Tracer sometimes fails to resolve Jaeger's address. In such a case run:
...
 ```
 and restart affected applications.
+
+## Testing
+Tests are based on [Arquillian Cube](http://arquillian.org/arquillian-cube/) which require an active connection to
+openshift cluster (via `oc`). Currently all templates are tested on minishift or local all-in-one cluster (`oc cluster
+up`).
+
+```bash
+minishift start // or oc cluster up, sometimes required oc login -u system:admin
```
Comment (objectiser) [inline README.md]: All works great - only comment is that I would move the ", sometimes ...." to a NOTE following the bash block, as current could be a bit confusing.

Comment (pavolloffay) [inline README.md]: I have removed it, It seems it's not necessary.

**Pull request merged**